support-frontend/assets/helpers/__tests__/validationTest.ts

```diff
@@ -0,0 +1,31 @@
+import { doesNotContainExtendedEmojiOrLeadingSpace } from 'pages/[countryGroupId]/validation';
+
+describe('validateRegex', () => {
+	it('leading space', () => {
+		expect(matchRegex(doesNotContainExtendedEmojiOrLeadingSpace, ' z')).toEqual(
```

Comment:
Super minor, but Jest has a [`toMatch` matcher](https://jestjs.io/docs/expect#tomatchregexp--string) which takes a regex, which I think could be really nice to use readability wise. So for example I think this one would become:

```
expect(' z').not.toMatch(doesNotContainExtendedEmojiOrLeadingSpace);
```

What do you reckon?

Reply:
Yes, looks more readable, hit a very suttle probelm with the positve `toMatch` that required the regex command to be explicitly declared as a (`RegEx`not a `string`). hopefully thats ok - example shown below->

		expect(' z').not.toMatch(
			new RegExp(doesNotContainExtendedEmojiOrLeadingSpace),
		);

Reply:
Oh right that's interesting, I wonder why? I guess you could create the `new RegExp` once at the top of the tests and re-use it?

Reply:
Yep have reused that now.